tests/integration/individual_response/test_individual_response.py

```diff
@@ -79,6 +79,22 @@ def _request_individual_response(self):
         self.post()
 
 
+class TestIndividualResponseHubDisabled(IndividualResponseTestCase):
+    def setUp(self):
+        super().setUp()
+        self.launchSurvey("test_individual_response_hub_disabled", region_code="GB-ENG")
+
+    def test_show_on_hub_false(self):
+        self._add_household_no_primary()
+
+        with self.assertRaises(AttributeError) as error:
+            individual_response_link = self.individual_response_link
+
+        self.assertEqual(
+            str(error.exception), "'NoneType' object has no attribute 'find_next'"
+        )
```

Comment:
Personally, I don't find this very clear on why this is happening. I think `self. individual_response_link` should return `None` when there isn't a link and this assert to be updated to `self.assertIsNone(self.individual_response_link)`. Thoughts?